imagebuildah/stage_executor.go

```diff
@@ -918,8 +919,14 @@ func (s *StageExecutor) Execute(ctx context.Context, base string) (imgID string,
 		// determining if a cached layer with the same build args already exists
 		// and that is done in the if block below.
 		if checkForLayers && step.Command != "arg" {
+			err = s.executor.localLayerProvider.PopulateLayer(ctx, s.builder.TopLayer)
+			if err != nil {
+				return "", nil, errors.Wrap(err, "error checking if cached image exists from a previous build")
```

Comment:
Please do not start error messages with "error"

Reply:
resolved